packages/domain/src/use-cases/transactions/BroadcastingError.ts

```diff
@@ -1,22 +1,6 @@
-import { EthereumAddress } from '@lens-protocol/shared-kernel';
+import { TransactionRequestModel } from '../../entities';
 
-import { Data } from './PayTransaction';
-
-export enum BroadcastingErrorReason {
-  /**
-   * The gas-less broadcasting of the tx was rejected, probably due to reaching a quota limit
-   */
-  REJECTED = 'REJECTED',
-  /**
-   * A not specified reason
-   */
-  UNSPECIFIED = 'UNSPECIFIED',
-}
```

Comment:
Turns out this is not really necessary at this level. All usage of this detail were quite a speculation, so removed it for now.